pkg/printers/internalversion/describe.go

```diff
@@ -1382,7 +1382,7 @@ func describeVolumeClaimTemplates(templates []api.PersistentVolumeClaim, w Prefi
 		printLabelsMultilineWithIndent(w, "  ", "Labels", "\t", pvc.Labels, sets.NewString())
 		printLabelsMultilineWithIndent(w, "  ", "Annotations", "\t", pvc.Annotations, sets.NewString())
 		if capacity, ok := pvc.Spec.Resources.Requests[api.ResourceStorage]; ok {
-			w.Write(LEVEL_1, "Capacity:\t%s\n", capacity)
+			w.Write(LEVEL_1, "Capacity:\t%s\n", capacity.String())
```

Comment:
`%v` will not work :(. Here's the output of `%v`:

```
  Capacity:	{{1073741824 0} {<nil>} 1Gi BinarySI}
```

When we print an object by `%s` in go, it'll call object's `String()`; but `Quantity.String()` is using pointer (for caching string), we have to call it explicitly.

Should be `func (q Quantity) String() string` not `func (q *Quantity) String() string` at https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/api/resource/quantity.go#L608 .